core/src/test/java/org/kiwix/kiwixmobile/core/dao/NewBookmarksDaoTest.kt

```diff
@@ -0,0 +1,162 @@
+/*
+ * Kiwix Android
+ * Copyright (c) 2020 Kiwix <android.kiwix.org>
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package org.kiwix.kiwixmobile.core.dao
+
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.mockkStatic
+import io.mockk.verify
+import io.objectbox.Box
+import io.objectbox.query.Query
+import io.objectbox.query.QueryBuilder
+import io.objectbox.rx.RxQuery
+import io.reactivex.Observable
+import org.junit.Test
+import org.kiwix.kiwixmobile.core.CoreApp
+import org.kiwix.kiwixmobile.core.bookmark.BookmarkItem
+import org.kiwix.kiwixmobile.core.dao.entities.BookmarkEntity
+import org.kiwix.kiwixmobile.core.dao.entities.BookmarkEntity_
+import org.kiwix.kiwixmobile.core.reader.ZimFileReader
+import org.kiwix.sharedFunctions.bookmarkItem
+import org.kiwix.sharedFunctions.zimFileReader
+
+internal class NewBookmarksDaoTest {
+
+  private val box: Box<BookmarkEntity> = mockk(relaxed = true)
+  private val newBookmarksDao = NewBookmarksDao(box)
+
+  @Test
+  fun `get bookmarks without current book`() {
+    mockkStatic(CoreApp::class)
+    every { CoreApp.getInstance().packageName } returns "pkg"
+    val zimFileReader: ZimFileReader = mockk()
+    val queryBuilder: QueryBuilder<BookmarkEntity> = mockk()
+    every { box.query() } returns queryBuilder
+    every { queryBuilder.order(BookmarkEntity_.bookmarkTitle) } returns queryBuilder
+    val query: Query<BookmarkEntity> = mockk(relaxed = true)
+    every { queryBuilder.build() } returns query
+    newBookmarksDao.getBookmarks(false, zimFileReader)
```

Comment:
there is no assertion happening, 

```
every { query.find() } returns bookmarkEntities
```
where `bookmarkEntities` is a list of maybe even just 1 item

Reply:
>there is no assertion happening

I want you to be verifying the result of `newBookmarksDao.getBookmarks(false, zimFileReader)` was my point